modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/connector/PetStoreExclusiveParameterRequiredWithNullExpressionTestCase.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.module.extension.connector;
+
+import org.junit.Test;
+import org.mule.test.module.extension.AbstractExtensionFunctionalTestCase;
+import org.mule.test.petstore.extension.Aquarium;
+import org.mule.test.petstore.extension.ExclusiveCashier;
+import org.mule.test.petstore.extension.ExclusivePetBreeder;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.Assert.fail;
+
+public class PetStoreExclusiveParameterRequiredWithNullExpressionTestCase extends AbstractExtensionFunctionalTestCase {
+
+  private final String TEST_VALUE = "TEST";
+
+  @Override
+  protected String getConfigFile() {
+    return "petstore-exclusive-parameters-required-with-null-expression.xml";
+  }
+
+  @Test
+  public void getBreederOperationFail() throws Exception {
+    try {
+      ExclusivePetBreeder exclusivePetBreeder = (ExclusivePetBreeder) flowRunner("getBreederOperation")
```

Comment:
this can be improved with `flowRunner("flowName").runExpectingException()` where you can put an Exception matcher to check the error message